ipython/fragment_reattachment_example.ipynb

```diff
@@ -0,0 +1,1102 @@
+    "                    amt = frag_amt[1]\n",
+    "                    frag1 = frag  # 1R\n",
+    "                    frag2 = species  # 2R1L\n",
+    "                    frag_new = FragList.merge_frag_to_frag(frag2, frag1, 'R')  # L,R,R\n",
+    "                    self.Rlist.append((frag_new, amt * 2))\n",
+    "                    self.RRLLlist.append(frag_new)\n",
+    "\n",
+    "            elif len(re.findall(r'R', species)) == 1 and len(re.findall(r'L', species)) == 2:\n",
+    "                paired_frag_list = FragList.pair_frag(amount, self.L1dict)\n",
+    "                for frag_amt in paired_frag_list:\n",
+    "                    frag = frag_amt[0]\n",
+    "                    amt = frag_amt[1]\n",
+    "                    frag1 = frag  # 1L\n",
+    "                    frag2 = species  # 1R2L\n",
+    "                    frag_new = FragList.merge_frag_to_frag(frag1, frag2, 'R')  # L,R,R\n",
+    "                    self.Llist.append((frag_new, amt * 2))\n",
+    "                    self.RRLLlist.append(frag_new)\n",
+    "\n",
+    "            elif len(re.findall(r'R', species)) == 3 and len(re.findall(r'L', species)) == 0:\n",
+    "                paired_frag_list = FragList.pair_frag(amount, self.L1dict)\n",
+    "                for frag_amt in paired_frag_list:\n",
+    "                    frag = frag_amt[0]\n",
+    "                    amt = frag_amt[1]\n",
+    "                    frag1 = frag  # 1L\n",
+    "                    frag2 = species  # 3R\n",
+    "                    frag_new = FragList.merge_frag_to_frag(frag1, frag2, 'R')  # L,R,R\n",
+    "                    self.Rlist.append((frag_new, amt * 2))\n",
+    "                    self.RRLLlist.append(frag_new)\n",
+    "\n",
+    "            # 3L\n",
+    "            elif len(re.findall(r'R', species)) == 0 and len(re.findall(r'L', species)) == 3:\n",
+    "                paired_frag_list = FragList.pair_frag(amount, self.R1dict)\n",
+    "                for frag_amt in paired_frag_list:\n",
+    "                    frag = frag_amt[0]\n",
+    "                    amt = frag_amt[1]\n",
+    "                    frag1 = frag  # 1R\n",
+    "                    frag2 = species  # 3L\n",
+    "                    frag_new = FragList.merge_frag_to_frag(frag2, frag1, 'R')  # L,R,R\n",
+    "                    self.Llist.append((frag_new, amt * 2))\n",
+    "                    self.RRLLlist.append(frag_new)\n",
+    "\n",
+    "    def update_lists(self):\n",
+    "        '''\n",
+    "        adds the 1-cuttinglabel fragments remaining after pairing 4- and 3-cutting label fragments to their corresponding list\n",
+    "        '''\n",
+    "        for one_R_frag, amt in self.R1dict.items():\n",
+    "            self.Rlist.append((one_R_frag, amt))\n",
+    "        for one_L_frag, amt in self.L1dict.items():\n",
+    "            self.Llist.append((one_L_frag, amt))\n",
+    "\n",
+    "    def grind(conc, size):\n",
```

Comment:
I think is one of the functions that used to be in `fragment_utils.py` (?), could you move it to a new `fragment_utils` file in the main rmg module and then import it here? One of our users reached out about missing it.